template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/.template.config/template.json

```diff
@@ -121,66 +121,66 @@
       "datatype": "string",
       "defaultValue": "https://login.microsoftonline.com/tfp/",
       "replaces": "https:////login.microsoftonline.com/tfp/",
-      "description": "The Azure Active Directory B2C instance to connect to (use with IndividualB2C auth type)."
+      "description": "The Azure Active Directory B2C instance to connect to (use with IndividualB2C auth)."
     },
     "SignUpSignInPolicyId": {
       "type": "parameter",
       "datatype": "string",
       "defaultValue": "",
       "replaces": "MySignUpSignInPolicyId",
-      "description": "The sign-in and sign-up policy ID for this project (use with IndividualB2C auth type)."
+      "description": "The sign-in and sign-up policy ID for this project (use with IndividualB2C auth)."
     },
     "ResetPasswordPolicyId": {
       "type": "parameter",
       "datatype": "string",
       "defaultValue": "",
       "replaces": "MyResetPasswordPolicyId",
-      "description": "The reset password policy ID for this project (use with IndividualB2C auth type)."
+      "description": "The reset password policy ID for this project (use with IndividualB2C auth)."
     },
     "EditProfilePolicyId": {
       "type": "parameter",
       "datatype": "string",
       "defaultValue": "",
       "replaces": "MyEditProfilePolicyId",
-      "description": "The edit profile policy ID for this project (use with IndividualB2C auth type)."
+      "description": "The edit profile policy ID for this project (use with IndividualB2C auth)."
     },
     "AADInstance": {
       "type": "parameter",
       "datatype": "string",
       "defaultValue": "https://login.microsoftonline.com/",
       "replaces": "https:////login.microsoftonline.com/",
-      "description": "The Azure Active Directory instance to connect to (use with Single-org or Multi-org auth types)."
+      "description": "The Azure Active Directory instance to connect to (use with SingleOrg or MultiOrg auth)."
     },
     "ClientId": {
       "type": "parameter",
       "datatype": "string",
       "replaces": "11111111-1111-1111-11111111111111111",
-      "description": "The Client ID for this project (use with individual and organizational auth types)."
+      "description": "The Client ID for this project (use with IndividualB2C, SingleOrg or MultiOrg auth)."
```

Comment:
Nit: I think we should use the Oxford comma here (https://en.wikipedia.org/wiki/Serial_comma), it's less ambiguous, e.g.:
`use with IndividualB2C, SingleOrg, or MultiOrg auth`
...adding a comma after "SingleOrg"
I don't know if our doc standards say anything about using or not using it.